examples/Feeds/feeds.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"qlova.org/seed/client"
+	"qlova.org/seed/client/clientside"
+	"qlova.org/seed/new/app"
+	"qlova.org/seed/new/button"
+	"qlova.org/seed/new/column"
+	"qlova.org/seed/new/feed"
+	"qlova.org/seed/new/row"
+	"qlova.org/seed/new/text"
+	"qlova.org/seed/new/textbox"
+	"qlova.org/seed/set"
+	"qlova.org/seed/use/css/units/vh"
+	"qlova.org/seed/use/js"
+)
+
+func main() {
+	newFruit := &clientside.String{}
+
+	values := []string{"Apples", "Pears"}
+	names := feed.With(feed.Go(func() []string {
+		return values
+	}))
+
+	app.New("Feeds",
+		column.New(
+			row.New(
+				set.MarginBottom(vh.New(1)),
+				text.New(text.Set("Fruit List"), text.SetSize(vh.New(3))),
+			),
+			row.New(
+				names.New(
+					row.New(
+						text.New(
+							text.SetStringTo(js.String{Value: names.Data.Value}),
+						),
+					),
+				),
+				client.OnLoad(names.Refresh()),
```

Comment:
So - the feed doesn't actually render the items without this line. However, even with this the feed still intermittently won't render the list on load.

Reply:
Yes, feeds don't refresh until you request them to.

Reply:
However it should load, that would be a bug if it isn't.